hasura.planx.uk/metadata/tables.yaml

```diff
@@ -246,6 +246,49 @@
         template_engine: Kriti
         url: '{{$base_url}}/webhooks/hasura/send-slack-notification'
         version: 2
+- table:
+    name: email_events_log
+    schema: public
+  select_permissions:
+    - role: demoUser
+      permission:
+        columns:
+          - retry
+          - response
+          - event_id
+          - event_type
+          - status
+          - created_at
+          - flow_id
+          - session_id
+        filter: {}
+      comment: ""
+    - role: platformAdmin
+      permission:
+        columns:
+          - retry
+          - response
+          - event_id
+          - event_type
+          - status
+          - created_at
+          - flow_id
+          - session_id
+        filter: {}
+      comment: ""
+    - role: teamEditor
+      permission:
+        columns:
+          - retry
+          - response
+          - event_id
+          - event_type
+          - status
+          - created_at
+          - flow_id
+          - session_id
+        filter: {}
+      comment: ""
```

Comment:
I matched "select" permissions to the `submission_services_log` view here - does this seem fine? Alternatively, can keep stricter until explicit plans to expose in editor. View does _not_ include any personal user data like email address or payment details, only session ID.

Reply:
Great question!

I'd favour not exposing it all until we need to - if it's intended to be for platform admins only lets capture that here 👍 

If / when we come to exposing it, we set the right permissions then.

I'll checkout the demoUser / `submission_services_log` permissions just now - I would have expected that to have the normal demo user filter of "I can only select my own sessions" applied here.

Reply:
PR here - https://github.com/theopensystemslab/planx-new/pull/4212

Reply:
Yep makes sense - will rollback & keep as `platformAdmin`-select only